traffic_editor/gui/editor.cpp

```diff
@@ -1432,29 +1468,35 @@ void Editor::populate_property_editor(const Vertex& vertex)
   const double scale = level.drawing_meters_per_pixel;
 
   property_editor->blockSignals(true);  // otherwise we get tons of callbacks
-  property_editor->setRowCount(5 + vertex.params.size());
+
+  // only get row entries relevant to properties, filter out those from human
+  std::map<QString, QString> row_entries;
+  const std::vector<std::string> human_vec =
+    HumanVtxPropTable::_required_components;
+  for (const auto& param : vertex.params)
+  {
+    if (std::find(human_vec.begin(), human_vec.end(),
+      param.first) == human_vec.end() )
+      row_entries[QString::fromStdString(param.first)] =
+        param.second.to_qstring();
+  }
```

Comment:
Nit, Generally not a big fan of using `std::find` in `std::vectors` since it will have to traverse the whole structure, if you need to find very often it might make more sense to have a unordered_set / unordered_map.